bazel-mode.el

```diff
@@ -76,7 +88,8 @@
   (setq-local comment-start-skip "#+")
   (setq-local comment-end "")
   (setq-local comment-use-syntax t)
-  (setq-local font-lock-defaults '(nil)))
+  (setq-local font-lock-defaults '(nil))
+  (add-hook 'before-save-hook 'bazel-mode--buildifier-before-save-hook))
```

Comment:
This changes the global hook. You either need to change the local hook or check for the mode in the hook function. Otherwise this will attempt to format every visited file with Buildifier.

Reply:
Done.